### ISSUE TITLE
Contact Center Review for Unified Appointment Experience - Pre-check-in, check-in - Updated designs with new screenshots

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#88274]
# Request the Contact Center Review 

## What this form is for
Use this template to request a Contact Center review of your product.

Please review the Self-service Product Guide Template [link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/self-service-product-guide-template.md) and the Contact Center Review How to [link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/request-contact-center-review.md)
 

## Description
*Please provide the name(s) of your product and a high-level description of your product.*

- Pre-Check-In: The Patient Check In (PCI) application workflow that occurs one to seven days before an appointment to collect Pre-Registration information. It is initiated after a Veteran confirms their appointment through an appointment reminder.
- Check In: The Patient Check In (PCI) application workflow that occurs within a window that is no more than 45 minutes before and up to 15 minutes after a Veteran's appointment. Online check in has logic to include pre-registration, checking in (which sets the VistA status to e-check-in: complete, and filing for travel reimbursement.

### Launch Date
*Please provide the launch or rollout date and details.*

~ early August

### Version Number
*Please provide the version number of the guide and/or video.*

- Pre-check-in 1.10
- Check-in 2.4

Feature notes: Design updates. No functional changes. Screenshots updated.

### Product Guide
*Please provide product guide link*

- [PRE-CHECK-IN_PRODUCT_GUIDE.docx](https://github.com/user-attachments/files/16200178/PRE-CHECK-IN_PRODUCT_GUIDE.docx)
- [PRE-CHECK-IN_PRODUCT_GUIDE.pdf](https://github.com/user-attachments/files/16200180/PRE-CHECK-IN_PRODUCT_GUIDE.pdf)
- [PATIENT.CHECK-IN_PRODUCT_GUIDE.docx](https://github.com/user-attachments/files/16200181/PATIENT.CHECK-IN_PRODUCT_GUIDE.docx)
- [PATIENT.CHECK-IN_PRODUCT_GUIDE.pdf](https://github.com/user-attachments/files/16200183/PATIENT.CHECK-IN_PRODUCT_GUIDE.pdf)

### Product Video
*Please provide product video link*

- [pre_check_in_demo.webm](https://github.com/user-attachments/assets/ddf6b779-8009-4495-b045-6b9f21bf82a4)
- [check_in_demo.webm](https://github.com/user-attachments/assets/63b86a65-b586-4806-a1ed-9f9b056e89f3)

### Product Video Transcript
*Please provide product video transcript link*

n/a

## Acceptance Criteria
- [ ] Platform: Contact Center QA has been completed 
- [ ] Platform: Contact Center sends artifacts to appropriate contact centers
- [ ] Platform: [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886) is updated

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Contact-center-review.1782317061.html) or the [Contact Center Review Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Contact-center-review.1782317061.html) on Platform website for more information about the Collaboration Cycle.

**Comment by starlow00** [comment]
Hi @benbrasso-agile6. In the pre-check-in guide, when I click the sample link (https://go.vetext.info/98jjjz1) at step 3 on page 4, I receive this warning in my Chrome browser on Mac. It may not be a cause for concern, but I wanted to raise it just in case:
<img width="1422" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMkRmQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--204c735cf7f8a3e50b1b9c43207741a36425689d/Screenshot%202024-07-15%20at%201.36.23%E2%80%AFPM.png" alt="Screenshot 2024-07-15 at 1.36.23 PM.png" />

**Comment by benbrasso-agile6** [comment]
@jwowk is this expected behavior ^ Or, I might have something incorrect. Of course, this was just a fake VEText link. So, that could be expected behavior.

**Comment by jwowk** [comment]
@benbrasso-agile6 yes, this might be leftover from development. The actual links come from https://www.va.gov/

**Comment by starlow00** [comment]
Thanks, @jwowk and @benbrasso-agile6. That being the case, how should I refer to the link in the guide? Should I qualify that it's a sample/development link, or should I leave out the specific link?

**Comment by benbrasso-agile6** [comment]
> @benbrasso-agile6 yes, this might be leftover from development. The actual links come from https://www.va.gov/

That's so funny; I thought it was a recent switch. Thanks for clearing that up.

@starlow00 Yes, we'll want to switch it to something like, https://www.va.gov/99aaaa1, which is just an example.

**Comment by jwowk** [comment]
One update, the prod ones include a /c/ before the unique code so it would look like this: https://www.va.gov/c/99aaaa1

**Comment by starlow00** [comment]
Got it, thanks @benbrasso-agile6 and @jwowk

**Comment by starlow00** [comment]
Pre-check-in and check-in drafts look good. Sending to VEO for their review.

**Comment by starlow00** [comment]
No changes from VEO. I have attached the current copies of the guides. Please use them for future revisions/additions/changes. Thank you!
[PRE-CHECK-IN_PRODUCT_GUIDE v1.10 SRT 07152024.docx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBelA3QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1bb22b5dc825759a28fc558d207775b7054f1c5a/PRE-CHECK-IN_PRODUCT_GUIDE%20v1.10%20SRT%2007152024.docx)

[PATIENT.CHECK-IN_PRODUCT_GUIDE v2.4 SRT 07162024.docx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBelQ3QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--851914b8fbcf61c2499e83635c6d53e6f0f1ef0a/PATIENT.CHECK-IN_PRODUCT_GUIDE%20v2.4%20SRT%2007162024.docx)

**Comment by benbrasso-agile6** [comment]
Thank you!
